ihmc-footstep-planning/src/main/java/us/ihmc/footstepPlanning/FootstepPlannerOutput.java

```diff
@@ -215,47 +215,51 @@ public void setSwingTrajectories(List<EnumMap<Axis3D, List<PolynomialReadOnly>>>
       this.swingTrajectories = swingTrajectories;
    }
 
-   public void setPacket(FootstepPlanningToolboxOutputStatus outputStatus)
+   /**
+    * This packs the parameter with all the values in the {@link FootstepPlanningToolboxOutputStatus} assuming things aren't null
+    * @param outputStatusToPack is the parameter that gets packed
+    */
+   public void setPacket(FootstepPlanningToolboxOutputStatus outputStatusToPack)
```

Comment:
Wanted to make this more clear that this gets packed, just trying to be more helpful for the user